app/src/main/java/com/mk/tasky/MainActivity.kt

```diff
@@ -20,20 +31,41 @@ class MainActivity : ComponentActivity() {
         setContent {
             TaskyTheme {
                 // A surface container using the 'background' color from the theme
-                Surface(
-                    modifier = Modifier.fillMaxSize(),
-                    color = MaterialTheme.colors.background
-                ) {
-                    LoginScreen()
-                }
+                Greeting()
             }
         }
     }
 }
 
 @Composable
 fun Greeting() {
-    LoginScreen()
+    Surface(
+        modifier = Modifier.fillMaxSize(),
+        color = Black
+    ) {
+        Column(modifier = Modifier.fillMaxSize()) {
+            Box(
+                modifier = Modifier.fillMaxSize().weight(1.5f),
+                contentAlignment = Alignment.Center
+            ) {
+                Text(
+                    text = "Welcome Back!",
+                    fontWeight = FontWeight.Bold,
+                    color = White,
+                    fontSize = 28.sp
+                )
+            }
+            Surface(
+                modifier = Modifier.fillMaxSize().weight(8.5f)
+                    .clip(RoundedCornerShape(topStart = 30.dp, topEnd = 30.dp)),
+                color = White
+            ) {
+                Box(modifier = Modifier.padding(16.dp)) {
```

Comment:
You could move this `padding()` call to the end of the modifier for the surface to get rid of this Box :)

Reply:
I tried it on both surfaces, yet I wasn´t able to replicate it